app/Commands/Know/Show.php

```
@@ -0,0 +1,179 @@
+            $this->displayFullEntry($knowledgeService, $entry);
+
+            return 0;
+
+        } catch (\Exception $e) {
+            $this->error("❌ Error showing knowledge: {$e->getMessage()}");
+
+            return 1;
+        }
+    }
+
+    private function displayFullEntry(KnowledgeService $knowledgeService, $entry): void
```

PHPMD (2.15.0)

Avoid unused parameters such as '$knowledgeService'. (Unused Code Rules)

(UnusedFormalParameter)